revolv/templates/base/partials/category_preferences.html

```diff
@@ -12,6 +13,6 @@
         </div>
     {% endfor %}
 
-    <div class="explanation-header">What's going on?</div>
-    <div class="explanation">Your donation to RE-volv has been invested in a revolving fund. Money from the fund is used to place solar equipment on community buildings. Over time, the community pays RE-volv back. These repayments are invested in even more solar projects. A fraction of the repayments from a solar investment originates from your investment. Your preferences directly affect where that chunk of money is invested.</div>
+    <div class="explanation-header">{{ settings.revolv_cms.DashboardSettings.category_preferences_explanation_header_text }}</div>
+    <div class="explanation">{{ settings.revolv_cms.DashboardSettings.category_preferences_explanation_text }}</div>
```

Comment:
This will need to have a `|safe` I believe - make sure you go through the process of editing it via the cms dashboard to see how it looks!